### ISSUE TITLE
[DEV] What are the different lootmodes ?

### ISSUE THREAD

**Issue by BarbzYHOOL** [azerothcore/forum#105]
http://www.azerothcore.org/wiki/loot_template

It's written:
Valid lootmodes include: 1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 16384, 32768.

Whare are they? what do they represent?

**Comment by Kitzunu** [comment]
They are to be assigned in a core script, that will switch to different loot modes.

For example in Ulduar you have Hardmode so if you do a specific tactic right it will trigger "hardmode" and thereby also triggering another lootmode allowing different kind of loot to drop

**Comment by BarbzYHOOL** [comment]
so it's 100% arbitrary and many of them are never used until we assign them to a custom hard mode script?

**Comment by Kitzunu** [comment]
I am 99% sure the hard modes don't work on AC which means they will never drop ^^
And it is not custom, it's just Blizzard being too dumb to invent Heroic difficulty sooner

**Comment by Kitzunu** [comment]
But I think only Ulduar uses them

**Comment by BarbzYHOOL** [comment]
Yes, hard mode doesn't work on AC from what I've read. (in ulduar)

I said custom because in our DB, the loot for hardmode are only from 1 to 32 or to 64. But in the wiki there are far more values but why??

I think there are some dungeons with hard mode too, example in Ahn Qiraj, depending on the order of killing some bosses, you get better loot

**Comment by Kitzunu** [comment]
(I dont know raids pre Wrath :P)

**Comment by h0lysp4nk** [comment]
Yeah, Temple of Ahn'Qiraj makes use of this feature in the bug trio encounter.
